examples/widgets_minimap.html

```diff
@@ -0,0 +1,129 @@
+            const debugMenu = new GuiTools('menuDiv', view);
+
+
+
+            // ---------- DISPLAY CONTEXTUAL DATA : ----------
+
+            // Add one imagery layer to the scene. This layer's properties are defined in a json file, but it could be
+            // defined as a plain js object. See `Layer` documentation for more info.
+            itowns.Fetcher.json('layers/JSONLayers/Ortho.json').then((config) => {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ColorLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            });
+
+            // Add two elevation layers, each with a different level of detail. Here again, each layer's properties are
+            // defined in a json file.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                view.addLayer(
+                    new itowns.ElevationLayer(config.id, config),
+                ).then(debugMenu.addLayerGUI.bind(debugMenu));
+            }
+            itowns.Fetcher.json('layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+
+            // ---------- ADD MINIMAP WIDGET : ----------
+
+            // Create a ColorLayer that shall be displayed on the minimap.
+            const minimapColorLayer = new itowns.ColorLayer('minimap', {
+                source: new itowns.WMTSSource({
+                    url: 'http://wxs.ign.fr/decouverte/geoportail/wmts',
+                    name: "ORTHOIMAGERY.ORTHOPHOTOS",
+                    format: 'image/jpeg',
+                    tileMatrixSet: 'PM',
+                    crs: 'EPSG:3857',
+                    extent: new itowns.Extent(
+                        'EPSG:3857',
+                        -20026376.39, 20026376.39,
+                        -20048966.10, 20048966.10
+                    ),
+                }),
+            });
+
+            // Create a minimap.
+            const minimap = new itowns_widgets.Minimap(view, minimapColorLayer, {
+                cursor: '+'
+            });
+
+
+
+            // ---------- ADD INTERACTION WITH MINIMAP : ----------
+
+            // When double-clicking the minimap, travel to the cursor location.
+            const cursorCoordinates = new itowns.Coordinates(minimap.view.referenceCrs);
+            const cursorPosition = new itowns.THREE.Vector3();
+            minimap.domElement.addEventListener('dblclick', (event) => {
+                if (minimap.view.getPickingPositionFromDepth(minimap.view.eventToViewCoords(event), cursorPosition)) {
+                    cursorCoordinates.setFromVector3(cursorPosition);
+                    view.controls.lookAtCoordinate({ coord: cursorCoordinates });
```

Comment:
could create a PR to add `pickCoordinates(mouseEvent or mousePosition)` and replace `getPickingPositionFromDepth`?

Reply:
Added in PR #1784.